src/_data/chipsets.yml

```diff
@@ -0,0 +1,30 @@
+- platform: 'Windows'
+  x86: '<span class="material-symbols" style="color: #158477;">verified</span>'
+  x64: '<span class="material-symbols" style="color: #158477;">verified</span>'
+  arm32: '<span class="material-symbols" style="color: #D43324">dangerous</span>'
+  arm64: '<span class="material-symbols" style="color: #13C2AD">gpp_maybe</span>'
+  risc-v: '<span class="material-symbols" style="color: #D43324">dangerous</span>'
```

Comment:
This is probably better off as the "N/A" icon. Windows for RISC-V simply doesn't exist right now.